### PR TITLE
Pushed down parent classes to remove extra labels on Neo4J graph

### PR DESCRIPTION
**Pull request by divyamadala30**

### CODE REVIEW

**Comment by divyamadala30** [comment]
Screen Shot 2021-02-11 at 9 32 43 AM

File: model/src/main/java/org/mskcc/cmo/metadb/model/MetaDbRequest.java [hunk 1]
```diff
@@ -1,11 +1,30 @@
...
+import org.neo4j.ogm.annotation.Id;
+import org.neo4j.ogm.annotation.NodeEntity;
+import org.neo4j.ogm.annotation.Relationship;
+
+/**
+ *
+ * @author ochoaa
+ */
+@NodeEntity
+public class MetaDbRequest {
```
Comment (ao508) [inline model/src/main/java/org/mskcc/cmo/metadb/model/MetaDbRequest.java:17]: still needs implements Serializable

File: model/src/main/java/org/mskcc/cmo/metadb/model/SampleManifestEntity.java [hunk 2]
```diff
@@ -1,10 +1,33 @@
...
-public class SampleMetadata implements Serializable {
+import org.mskcc.cmo.metadb.model.converter.LibrariesStringConverter;
+import org.mskcc.cmo.metadb.model.converter.QcReportsStringConverter;
+import org.neo4j.ogm.annotation.GeneratedValue;
+import org.neo4j.ogm.annotation.Id;
+import org.neo4j.ogm.annotation.NodeEntity;
+import org.neo4j.ogm.annotation.typeconversion.Convert;
+
+@NodeEntity
+public class SampleManifestEntity {
```
Comment (ao508) [inline model/src/main/java/org/mskcc/cmo/metadb/model/SampleManifestEntity.java:15]: still needs implements Serializable

**Review by ao508 — CHANGES_REQUESTED** [review]
2 classes still need the implements serializable, I've left a comment on both

**Review by ao508 — APPROVED** [review]

**Pull request merged**